### PR TITLE
Implemented `PutMetadataSlice`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #227 
Implemented `PutMetadataSlice` and added tests for it. Also, made `DeployMetadata` synchronous, it's more convenient this way.

### CODE REVIEW

File: src/main/java/com/artipie/maven/http/PutMetadataSlice.java [hunk 1]
```diff
@@ -56,6 +67,30 @@ public PutMetadataSlice(final Storage asto) {
...
+                        new Key.From(
+                            UpdateMavenSlice.TEMP,
+                            new KeyFromPath(matcher.group("pkg")).string(),
+                            new DeployMetadata(xml).release(),
+                            "maven-metadata.xml"
+                        ),
+                        new ContentWithSize(
+                            new Content.From(xml.getBytes(StandardCharsets.US_ASCII)),
+                            headers
+                        )
```
Comment (g4s8) [inline src/main/java/com/artipie/maven/http/PutMetadataSlice.java:87]: @olenagerasimova it's not required, `new Content.From(byte[])` constructor creates content with size

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check one comment

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**